lib/rules/require-valid-alt-text.js

```diff
@@ -126,7 +157,10 @@ module.exports = class RequireValidAltText extends Rule {
             }
           }
         } else if (isInput) {
-          const isImageInput = AstNodeInfo.hasAttributeValue(node, 'type', 'image');
+          const isImageInput = NodeMatcher.match(node, {
+            attributes: [{ name: 'type', value: { type: 'TextNode', chars: 'image' } }],
+          });
+          // const isImageInput = AstNodeInfo.hasAttributeValue(node, 'type', 'image');
```

Comment:
```suggestion
```

Reply:
👍 thank you **so much** for catching that!